vhdbuilder/scripts/automate_version_bump.sh

```diff
@@ -57,11 +58,11 @@ update_image_version() {
 }
 
 create_image_bump_pr() {
-    create_branch $BRANCH_NAME
+    create_branch $BRANCH_NAME $PR_TARGET_BRANCH
     update_image_version
 
     set +x
-    create_pull_request $NEW_IMAGE_VERSION $GITHUB_PAT $BRANCH_NAME $PR_TITLE
+    create_pull_request $NEW_IMAGE_VERSION $GITHUB_PAT $BRANCH_NAME $PR_TARGET_BRANCH $PR_TITLE
```

Comment:
nit: put the variables inside quotes just in case they have a space in the values. That way bash won't split the strings and pass the wrong args to create_pull_request.

I know that most of those values can't have spaces, so it's a very minor nit.

Reply:
yep that's fair, I think we should more strictly lint our bash scripts in general (until we're fully in go :))

I'd opt for taking care of this in a separate PR we merge into dev then cherry-pick to master however